pkg/health/heartbeat/heartbeat.go

```diff
@@ -112,6 +112,8 @@ func (hb *Heartbeat) RenewMemberLease(ctx context.Context) error {
 	}
 
 	memberID := strconv.FormatUint(response.Header.GetMemberId(), 16)
+	memberID += ":"
+	memberID += strconv.FormatUint(response.Header.GetClusterId(), 16)
```

Comment:
change the name to `memberInfo` or something more relevant, instead of `memberID` as it now no longer contains only the memberID.